codeforboston/maple
# Updated Notification behaviour

# Summary

- Based off of last week's discussion:
  - The notifications have been split between BillHistoryUpdateNotification and TestimonySubmitNotification
  - These notifications will have flags that will determine which notifications to display when it's filtered on the Newsfeed.
  - Added an position icon to the testimony notifications

# Checklist


# Screenshots

<img width="1512" alt="Screen Shot 2024-09-24 at 7 16 16 PM" src="https://github.com/user-attachments/assets/59416645-7f6a-4cd1-9059-b90116d18178">


# Known issues

_If you've run against limitations or caveats, include them here. Include follow-up issues as well._

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Follow a bill or organization
2. Have a person/organization post on the bill
3. Check newsfeed to see if it's populated properly
